### DISCUSSION TITLE
23.10.19 데일리 스크럼

### DISCUSSION THREAD

**Discussion by kdmstj** [petit-a-petit/MIML#34] [Daily Scrum]
.

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
  - DTO<->Entity 처리(Service 계층 결정)
  - 관련 테스트 코드 작성
- 내일은 어떤 일을 하나요?
  - 꼭 PR 올리기 
  - 스포티파이 노래 API 사용 확인

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 트랜잭션 반대 상황 검증
- 내일은 어떤 일을 하나요?
  - 비동기 작업 검증
  - PR 작성

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 리다이렉트 에러 해결 과정 정리
    - 코드리뷰 반영중..
- 내일은 어떤 일을 하나요?
    - 코드 수정 후 다시 PR

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 트랜잭션 에러 수정 완료 😋
* 내일은 어떤 일을 하나요?
  * 발표 준비 및 코테 준비
